packages/editor/src/components/post-featured-image/index.js

```diff
@@ -32,11 +32,20 @@ function PostFeaturedImage( { currentPostId, featuredImageId, onUpdateImage, onR
 	let mediaWidth, mediaHeight, mediaSourceUrl;
 	if ( media ) {
 		const mediaSize = applyFilters( 'editor.PostFeaturedImage.imageSize', 'post-thumbnail', media.id, currentPostId );
+		const fallbackMediaSize = applyFilters( 'editor.PostFeaturedImage.fallbackImageSize', 'thumbnail', media.id, currentPostId );
```

Comment:
Do we really need a new filter for the fallback? Can't we use the existing filter e.g:
```
const fallbackMediaSize = applyFilters( 'editor.PostFeaturedImage. imageSize', 'thumbnail', media.id, currentPostId );
````
Another thing to consider is that the filters used here may have complex logic it would be nice to only compute the fallbackMediaSize if the mediaSize does not exist.

Reply:
@jorgefilipecosta 

I implemented you suggestions in the latest commit.

- The fallback size now uses the same filter as the default size.
- The fallback size now get computed only when default size is missing